### ISSUE TITLE
Main menu appearance on win32.

### ISSUE THREAD

**Issue by Shadowblitz16** [modern-forms/Modern.Forms#82]
What is your opinion on this?

![image](https://github.com/user-attachments/assets/26759aac-0c2c-42c5-bb04-ff4036d3cd9d)

Winfroms uses custom rendered menus while win32 menus don't

Winforms menus are not native looking win32 ones are.

**Comment by jpobst** [comment]
As a fully not-native drawn interface, I think the menus will look like `MenuStrip`.  This means they (and all other controls) will look the same on all platforms.  They will not look like native controls.

**Comment by Shadowblitz16** [comment]
> As a fully not-native drawn interface, I think the menus will look like `MenuStrip`. This means they (and all other controls) will look the same on all platforms. They will not look like native controls.

That's too bad that completely removes the point for me to use this library.

**Comment by sgf** [comment]
> That's too bad that completely removes the point for me to use this library.

No, the meaning of this library itself is not to get the look of win32. Rather, the benefits of reducing platform dependencies (cross-platform), removing handle dependencies and eliminating the need for win32 handles are obvious, and controls will be more flexible than ever.

**Comment by Shadowblitz16** [comment]
I don't want my apps looking like wpf/avalonia and I don't want them to look like windows fluent design garbage.
If your going to do custom drawing look into making modern-forms look like something simular to java breeze's flatlaf.

No one is going to use your library to do lookless controls when there is avalonia, what we really need is cross platform traditional desktop toolkit.

This mean either using native desktop controls on window, mac and linux or doing something like jetbrains did and use traditional desktop controls and theme java swing and flatlaf

**Comment by jpobst** [comment]
One of the design goals is for Modern.Forms to be pretty theme-able.  Flatlaf already looks pretty similar to what Modern.Forms currently ships.  It doesn't look like it would be hard for someone to create a theme to look like it.
